example-pipelines/environments/prod/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+project_id = "PROJECT_ID"
+vpc_network_name = "example-vpc"
```

Comment:
prod and dev vpc_names are same which will cause an error. 
Also will need the vm name to be parameterized to avoid conflict in prod and dev

Reply:
updated. I kind of "assumed" that the prod and dev projects will be different but they are the same for our example. Changed VPC name and network CIDR as well.

Reply:
removed two un-used tfvars as well.